docs/userguide/execution.rst

```diff
@@ -48,7 +48,8 @@ Parsl currently supports the following providers:
 8. `parsl.providers.GoogleCloudProvider`: This provider allows you to provision and manage cloud nodes from Google Cloud.
 9. `parsl.providers.KubernetesProvider`: This provider allows you to provision and manage containers on a Kubernetes cluster.
 10. `parsl.providers.AdHocProvider`: This provider allows you manage execution over a collection of nodes to form an ad-hoc cluster.
-11. `parsl.providers.LSFProvider`: This provider allows you to schedule resources via IBM's LSF scheduler
+11. `parsl.providers.LSFProvider`: This provider allows you to schedule resources via IBM's LSF scheduler.
```

Comment:
It's good to avoid making other incidental changes that aren't part of the issue you are working on - for example, your PR title is about notices on the CobaltProvider, but this change here is nothing to do with the cobalt provider! I'll put a new title on the PR that is a bit more broad.

Reply:
I am really sorry for that. I am a rookie. Thank you for the feedback.